content/en/docs/developer_guide/project/configuration/project_configuration.md

```diff
@@ -34,7 +34,7 @@ The following topics describe the project configuration settings.
 
 | Key | Type | Default | Description |
 | --- | --- | --- | --- |
-| disabled | Boolean | false | Set to `true` to display the generated OpenAPI Docs. Changing the setting only works in production. Swagger documentation is always available in dev mode. |
+| disabled | Boolean | false | **Deprecated {{% deprecation/link D054 %}}**. Set to `true` to display the generated OpenAPI Docs. Changing the setting only works in production. Swagger documentation is always available in dev mode. |
```

Comment:
@team We said we might just completely hide this instead of saying it is deprecated in the docs. Thoughts?

Reply:
I was iffy about it and thought the same, but there were others like this.  But y, I think I recall that conversation and would agree it should just get deleted.  It doesn't "work" as a flag any more, so no point having it in docs.